### PR TITLE
Text corrections

### PR DESCRIPTION
**Pull request by criadoperez**
Corrected several typos

### CODE REVIEW

**Comment by paperdigits** [comment]
Thank you so much for making these corrections!

"Tread carefully" is actually correct (in content/module-reference/processing-modules/diffuse.md), if you can revert back that one phrase, everything else looks great.

File: content/module-reference/processing-modules/diffuse.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ All speeds are added (first to fourth orders), and the sums "`first order + seco
 
 ## scale
 
-Natural diffusion is supposed to happen only to the closest neighbouring coordinates. That is, at each iteration, each pixel should only interact with its 9 nearest neighours.
```
Comment (elstoc) [inline content/module-reference/processing-modules/diffuse.md:71]: Much as it pains me, this should use the American spelling - so "neighbors"

File: content/module-reference/processing-modules/diffuse.md [hunk 2]
```diff
@@ -172,7 +172,7 @@ These two steps can be performed on the zoomed-out image. Remember that, while g
 
 At this point, you may want to tweak the edge sensitivity to take care of any edge artifacts. In theory, diffusing in the isophote direction ensures that diffusion is contained inside edges, but this is not sufficient when corners and sharp convex shapes are present in the image.
 
-When the edge sensitivity control has been adjusted to produce satisfying results, the image usually becomes quite soft. In most cases it will be necessary, at this point, to increase the number of iterations in order to compensate. This will come with a performance penalty so tread carefully with the performance/quality trade-off depending on your hardware. If you can't increase the number of iterations, you will have to increase the diffusing speed.
```
Comment (elstoc) [inline content/module-reference/processing-modules/diffuse.md:175]: "tread carefully" is correct here

**Review by elstoc — COMMENTED** [review]
Just a couple of minor corrections. Otherwise, looks good

**Comment by criadoperez** [comment]
reverted the `tread carefully`

File: content/module-reference/processing-modules/diffuse.md (hunk 2 above)
Comment (criadoperez) [reply content/module-reference/processing-modules/diffuse.md]: reverted it back. thanks.

File: content/module-reference/processing-modules/diffuse.md (hunk 1 above)
Comment (criadoperez) [reply content/module-reference/processing-modules/diffuse.md:71]: Ok. Changed to "neighbors". But in this case, to be consistent I also changed the "neihbouring coordinates" to "neighboring corrdinates", to be consistent.

**Comment by elstoc** [comment]
Nice one. Thanks!

**Pull request merged**